src/Sentry/Internal/SparseArray.cs

```diff
@@ -34,6 +36,6 @@ public SparseScalarArray(T uninitializedValue, int capacity = 0)
 
     public bool ContainsKey(int key)
     {
-        return key > 0 && key < _items.Count && !_uninitializedValue.Equals(_items[key]);
+        return key >= 0 && key < _items.Count && !_uninitializedValue.Equals(_items[key]);
```

Comment:
Is this bug int he original implementation?

Reply:
yes and there were no tests (fixed that as well of course)